src/features/geography/components/GeographyMap/index.tsx

```diff
@@ -138,7 +151,13 @@ const GeographyMap: FC<MapProps> = ({ areas }) => {
           width: '100%',
         }}
       >
-        <Box display="flex" justifyContent="space-between" px={2} py={1}>
+        <Box
+          ref={(ref) => (containerElm.current = ref as HTMLElement)}
```

Comment:
Would it be possible to just do this?

```suggestion
          ref={containerElm}
```